resources/views/formfields/relationship.blade.php

```diff
@@ -26,10 +26,13 @@
 
             @else
 
-                <select class="form-control select2" name="{{ $options->column }}">
+                <select class="form-control select2-ajax" name="{{ $options->column }}"
+                data-get-items-route="{{route('voyager.' . $dataType->slug.'.relation')}}"
+				data-get-items-field="{{$row->field}}"
+                >
                     @php
                         $model = app($options->model);
-                        $query = $model::all();
+						$query = $model::where($options->key,$dataTypeContent->{$options->column})->get();
```

Comment:
Please watch your tabs vs spaces.  This now looks ridiculous because you introduced tabs when everything else uses spaces.